.circleci/config.yml

```diff
@@ -10,6 +10,7 @@ executors:
     environment:
         JVM_MEMORY: 3500M # Default JVM maximum heap limit
         LANG: en_US.UTF-8 # required by sbt when it sees boost directories
+        MARSHAL_JLEVEL: 8 # This is unset by default, leading to starvation conditions on the manager
```

Comment:
It would be really nice if `marshal` could make this default to `multiprocessing.cpu_count()` (or perhaps some multiple of that) rather than defaulting to unrestricted `-j` because that's usually bad juju.

Reply:
@NathanTP thoughts?

Reply:
Good timing, I actually fixed that last week: firesim/firemarshal#207
I'm just waiting on someone to approve it.